src/runtime/particle-execution-context.ts

```diff
@@ -169,6 +169,11 @@ export class ParticleExecutionContext implements StorageCommunicationEndpointPro
           }
         }
         message.id = id;
+        if (entityId != null) {
```

Comment:
Switching on entityId seems a little strange. Might be worth saying something about that in a comment on this `onProxyMessage` function.

Reply:
New comment above the if statement. What do you think?

Reply:
Looks good, thanks!